addons/textalog/nodes/dialogue_base_node/portraits_node/portraits_node.gd

```diff
@@ -105,7 +111,11 @@ func add_portrait(
 	
 	grab_portrait_focus(_texture_rect)
 	
-	emit_signal("portrait_added", character, portrait)
+	if changed:
+		emit_signal("portrait_changed", character, portrait)
```

Comment:
This can lead a problem since `portrait_changed` signal is emmited by `change_portrait` function.